hera_cal/abscal.py

```diff
@@ -83,28 +81,32 @@ def __init__(self, model, data, refant=None, wgts=None, antpos=None, freqs=None,
 
         Parameters:
         -----------
-        model : visibility data of refence model, type=DataContainer
+        model : Visibility data of refence model, type=dictionary or DataContainer
                 keys are antenna-pair + polarization tuples, Ex. (1, 2, 'xx').
                 values are complex ndarray visibilities.
                 these must be 2D arrays, with [0] axis indexing time
                 and [1] axis indexing frequency.
 
-                Optionally, model can be a path to a miriad or uvfits file, or a
-                pyuvdata.UVData object, or a list of either.
+                Optionally, model can be a path to a miriad file, a
+                pyuvdata.UVData object or hera_cal.HERAData object,
+                or a list of either.
 
-        data : visibility data of measurements, type=DataContainer
-               keys are antenna pair + pol tuples (must match model), values are
-               complex ndarray visibilities matching shape of model
+        data :  Visibility data, type=dictionary or DataContainer
+                keys are antenna-pair + polarization tuples, Ex. (1, 2, 'xx').
+                values are complex ndarray visibilities.
+                these must be 2D arrays, with [0] axis indexing time
+                and [1] axis indexing frequency.
 
-                Optionally, model can be a path to a miriad or uvfits file, or a
-                pyuvdata.UVData object, or a list of either. In this case, wgts, antpos,
-                freqs, times and pols are overwritten with equivalent information from data object.
+                Optionally, data can be a path to a miriad file, a
```

Comment:
Do we want to support uvh5 rather than miriad? Or just leave this as a legacy interface?

Reply:
I added a filetype kwarg to support both

Reply:
Where? I don't see it.